### PR TITLE
Adding date

### PR DESCRIPTION
**Pull request by sanu29**
Search Bar working
Due Date added 
Sorting with Date added

### CODE REVIEW

**Review by ritu9898 — APPROVED** [review]
The code looks readable and good to me.

File: src/components/EditNote.jsx [hunk 1]
```diff
@@ -8,72 +10,93 @@ export function EditNote(note) {
...
+                            <textarea className=" para6 text-align-justify w-100 form-text margin-none bg-white font-color-dark" placeholder="Content"
+                            onChange={(e)=>setContent(e.target.value)}  value={content}
+                            ></textarea>
+                    
+                        <div className="d-flex w-100 justify-content-evenly align-items-center">
+                          <div className="d-flex w-100 justify-content-start align-items-center">
+                            <div className="position-relative"onClick={
+                                ()=>{
+                                    if(dispDateAndTime==="none"){
+                                    setDIspDateAndTime('block');
```
Comment (sanjitsarkar) [inline src/components/EditNote.jsx:54]: Correct variable naming

**Review by sanjitsarkar — APPROVED** [review]
Rest looks good

**Review by srishtigautam21 — APPROVED** [review]
The code is clean and understandable
Good to merge

**Pull request merged**